logbook-parent/pom.xml

```diff
@@ -52,7 +52,7 @@
         <spring-security.version>5.8.1</spring-security.version>
 
         <junit.version>5.9.2</junit.version>
-        <mockito.version>4.11.0</mockito.version>
+        <mockito.version>5.1.1</mockito.version>
```

Comment:
We can not jump to Mockito 5 without updating to java 11.
https://github.com/mockito/mockito/releases/tag/v5.0.0

Reply:
https://github.com/zalando/logbook/pull/1375 requires java 17 anyway

Reply:
Tests now pass due to upgrade to java compiler version 17 (#1434).